src/components/Login/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react'
+import { useLocation, Link } from 'wouter'
+import useUser from '../../hooks/useUser'
+import Eye from '../Icons/Eye'
+import IMAGES from '../../images/images'
+
+export default function Login () {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [showPassword, setShowPassword] = useState(false);
+  const [, navigate] = useLocation()
+  const {loginError, isLogged, login, loginErrorMessage} = useUser()
+
+  useEffect(() => {
+    if (isLogged) {
+      navigate('/')
+    }
+  }, [isLogged, navigate])
+
+  const submitHandler = (e) => {
+    e.preventDefault()
+    login({ email, password })
+  }
+
+  const changeEmailHandler = (e) => {
+    setEmail(e.target.value)
+  }
+
+  const changePasswordHandler = (e) => {
+    setPassword(e.target.value)
+  }
+
+  const handlePasswordToggle = () => {
+    if(showPassword){
+      setShowPassword(false)
+    } else {
+      setShowPassword(true)
+    }
+  }
+
+  return (
+    <div className="container">
+      <img src={IMAGES.union} alt="Black market logo" />
+      <form onSubmit={submitHandler}>
+        <div className="email">
+          <label htmlFor="email">Email</label>
+          <input
```

Comment:
it would be a good idea to create a custom component for buttons and inputs too! since this are gonna be used in all the app, and they look pretty much the same!

Reply:
a good example could be an input that contains a label thats optional, this way its gonna be way easier to use in the future, and instead of creating different styles for every input, you have the styles defined in the component